scripts/beforeInstall.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function(ctx) {
+    var exec = require('child_process').execSync;
+    var Q = ctx.requireCordovaModule('q');
+    var deferral = new Q.defer();
+
+    console.log("Detecting cordova version...");
+    var cordovaV = ctx.opts.cordova.version;    
+    console.log("Cordova version is " + cordovaV);
+
+    var plugins = exec('cordova plugin ls');
+    if (plugins.includes('cordova-plugin-file-transfer')) {
+        console.log("Removing file transfer plugin to ensure cordova compatibility... ");    
+        exec('cordova plugin remove cordova-plugin-file-transfer');
```

Comment:
@annakocheshkova, I'm not sure if it's a good decision to remove user's installed plugins via script automatically, could you please investigate how Cordova resolve dependencies while installing plugins? I think we should provide only a logic that adds new plugins and avoid of logic that removes user's defined dependencies.